images-list

```diff
@@ -444,6 +444,9 @@ longhornio/longhorn-ui rancher/mirrored-longhornio-longhorn-ui v1.2.4
 messagebird/sachet rancher/mirrored-messagebird-sachet 0.2.3
 messagebird/sachet rancher/mirrored-messagebird-sachet 0.2.6
 minio/minio rancher/mirrored-minio-minio RELEASE.2020-07-13T18-09-56Z
+neuvector/controller rancher/mirrored-neuvector-controller 5.0.0-b1
+neuvector/enforcer rancher/mirrored-neuvector-enforcer 5.0.0-b1
+neuvector/manager rancher/mirrored-neuvector-manager 5.0.0-b1
```

Comment:
Is b1 the final release ? We only mirror actual releases, and not rc's/alphas, unless its special situation. Otherwise looks fine.